library/packages/src/modules/Package.rb

```diff
@@ -512,6 +532,24 @@ def find_backend(target)
 
       found_backend || backend
     end
+
+    # checks if working on transactional system
+    # if so, then it shows popup to user and abort yast
+    def check_transactional_system!(packages, mode = :install)
+      return unless IsTransactionalSystem()
+
+      msg = _("Transactional system detected. ")
+      case mode
+      when :install then msg += _("Following packages have to be installed manually:")
```

Comment:
I know I'm late to the party but... https://dictionary.cambridge.org/grammar/british-grammar/following-or-the-following

Not to mention the usage of plural when there is only one package. :wink: